src/s1reader/s1_reader.py

```diff
@@ -569,11 +570,9 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
         tree_lads = ET.parse(f_lads)
         product_annotation = ProductAnnotation.from_et(tree_lads)
 
+        rfi_annotation_path = annotation_path.replace('annotation/', 'annotation/rfi/rfi-')
         # Load RFI information if available
-        if ipf_version >= RFI_INFO_AVAILABLE_FROM:
-            rfi_annotation_path =\
-                    annotation_path.replace('annotation/',
-                                            'annotation/rfi/rfi-')
+        if Path(rfi_annotation_path).exists() and ipf_version >= RFI_INFO_AVAILABLE_FROM:
```

Comment:
I have a suggestion and a question:
- I think it would be a good idea to print out the warning message that RFI annotation was not found, so the reader is skipping RFI info loading.
- What's the difference between `os.path.exists(file_path)` and `Path(file_path).exists()`?

Reply:
1. sure thing, I'll add that logging message
2. No difference- I've just started to find the `pathlib` API to be nicer to use, but i can change that if you want consistency

Reply:
Thanks for the response @scottstanie. In general I would like to suggest to import module as little as it can, even though it's included in python distribution.

>There should be one-- and preferably only one --obvious way to do it.
>  -- PEP20 - The Zen of Python

Reply:
Interesting.
I'm not sure in what case we are going to have `/` at the end of the file path like what is in the example above. Can you  give me some example?

Reply:
I was going this in Ipython where it autocompleted the file name, but in this case it was the SAFE directory (so it added a `'/'`). it was unexpected that a function to download the orbit would work/fail depending on if you include a trailing slash to the SAFE directory name

Reply:
I see. Interesting. That being case, I agree to switch to using `Path`